### ISSUE TITLE
[EN][Quest][Mount Hyjal][Firelands Invasion]The Protectors of Hyjal(ID 29128)

### ISSUE THREAD

**Issue by Aival94** [Atlantiss/BugTracker#5488]
Daily quest is missing.

Should be available after completing quest [Through the Gates of Hell](http://www.wowhead.com/quest=29201/through-the-gates-of-hell)

Wowhead Link: [The Protectors of Hyjal](http://www.wowhead.com/quest=29128/the-protectors-of-hyjal)
Wowhead Link: [Molten Front](http://www.wowhead.com/guide=58/a-guide-to-the-molten-front-and-respective-quests)

**Comment by Aival94** [comment]
Ok.Quest is now normally available.Probably was related to http://community.atlantiss.eu/index.php?/topic/6956-npc53073-captain-soren-moonfall/#entry25164 so if the order is ok i think you can close this bug report.

**Comment by Thefakemccoy** [comment]
This quest is still bugged, Not missing but. Missing the elite group of fighters.

I have selected an elite group of fighters to assist you. Travel to Sethria's Roost. Your compatriots have been instructed to meet you once you arrive. Then put an end to the elementals' assault.

**Comment by Aival94** [comment]
Yeah is true that the 4 NPCs are not spawned to help so it harder to be done but mobs there are not using any ability at all as well.If Bildo not open it so i edit it to a bug report or tell me if need to report it as might be already reported i will make an other report i guess.

**Comment by Thefakemccoy** [comment]
Its not reported not a open one. Only the achv for this with dueling.
